### DISCUSSION TITLE
BU KADAR IBAX  BANA NEREDEN GELDİ? :)

### DISCUSSION THREAD

**Discussion by hzbthn** [IBAX-io/go-ibax#1214] [Bug Submission]
![image](https://user-images.githubusercontent.com/94884847/151529907-9cb0e914-0299-4665-b156-c3414636e25b.png)

**Comment by cobra006** [comment]
GÜZEL KAZANMIŞSINIZ

**Reply by Stendardo17** [reply]
bende de aynı durum mevcut
Bakiye : 9999.994345 IBXC

**Comment by metinx33** [comment]
balance yazan yerde bendede baya var önemli olan deposit yazan yerde artı rakamın olması. yada üzümünü ye bagını sorma :)

**Comment by huseyin03** [comment]
> ![resim](https://user-images.githubusercontent.com/94884847/151529907-9cb0e914-0299-4665-b156-c3414636e25b.png)

bildiğim kadarıyla onları kullanamıcaz deneme için falan verildi diyorlardı

**Comment by erhanqwerty** [comment]
test ağı olduğu içindir

**Comment by sedat2855** [comment]
TEST İÇİNDİR
